src/Bicep.Core.Samples/Files/baselines/ResourceDerivedTypes_LF/main.formatted.bicep

```diff
@@ -0,0 +1,16 @@
+type foo = resource < 'Microsoft.Storage/storageAccounts@2023-01-01' >
```

Comment:
Looks like there's a formatting issue here

Reply:
I made sure this was working correctly with the v2 pretty printer but completely forgot about the non-experimental one! Corrected in latest.